site/assets/scss/_content.scss

```diff
@@ -104,3 +112,31 @@
     border-left: var(--#{$prefix}border-width) solid $border-color;
   }
 }
+
+// Boosted mod: no `.bd-summary-link`
+
+// scss-docs-start custom-color-mode
+[data-bs-theme="blue"] {
+  --bs-body-color: var(--bs-white);
+  --bs-body-color-rgb: #{to-rgb($white)};
+  --bs-body-bg: var(--bs-blue);
+  --bs-body-bg-rgb: #{to-rgb($blue)};
+  --bs-tertiary-bg: #{$blue-600};
+
+  .dropdown-menu {
+    --bs-dropdown-bg: #{mix($blue-500, $blue-600)};
+    --bs-dropdown-link-active-bg: #{$blue-700};
+  }
+
+  .btn-secondary {
```

Comment:
Doesn't behave like the Boosted one but might be fine as-is ?

Reply:
Tried something via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757#discussion_r1065635732. You'll tell me what you think.